Cloudinary/Classes/Core/Features/UploadWidget/WidgetViewControllers/CLDWidgetPreviewViewController.swift

```diff
@@ -35,14 +37,17 @@ class CLDWidgetPreviewViewController: UIViewController {
     private(set)  var collectionView: UICollectionView!
     private(set)  var mainImageView : UIImageView!
     private(set)  var uploadButton  : UIButton!
-    private(set)  var images        : [CLDWidgetImageContainer]
+    
+    private(set)  var videoView     : CLDVideoView!
+    
+    private(set)  var assets        : [CLDWidgetAssetContainer]
     internal weak var delegate      : CLDWidgetPreviewDelegate?
     private(set)  var selectedIndex : Int
-    
+        
     // MARK: - init
-    init(images: [CLDWidgetImageContainer], delegate: CLDWidgetPreviewDelegate? = nil) {
+    init(assets: [CLDWidgetAssetContainer], delegate: CLDWidgetPreviewDelegate? = nil) {
```

Comment:
We need to make sure another constructor exists that can handle the original type, otherwise it's a breaking change.